pages/workspaces/[workspaceId]/index.tsx

```diff
@@ -50,15 +52,22 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   setCookie({ response: context.res, name: WORKSPACE_ID_COOKIE_NAME, value: workspaceId });
 
-  return { props: { workspace: data } };
+  const ogImage = new URL(
+    getWorkspaceOgImage(data as Workspace, range),
+    process.env.NEXT_PUBLIC_BASE_URL || "http://localhost:3000"
```

Comment:
I know this is for dev, but I'd be explicit with the check for production URL. 😅

Reply:
👍  What's the best way to check?

Reply:
I have this also

```ts
  const host = context.req.headers.host?.includes("localhost")
    ? `http://${context.req.headers.host}`
    : process.env.NEXT_PUBLIC_BASE_URL;
  const ogImage = new URL(getWorkspaceOgImage(data as Workspace, range), host);
```

Reply:
Actually the env var is undefined locally, so I think we're good.